package.json

```diff
@@ -9,69 +9,43 @@
   },
   "type": "module",
   "engines": {
-    "node": ">=16"
+    "node": ">=18"
   },
   "dependencies": {
     "@bufbuild/buf": "1.50.0",
     "@bufbuild/protobuf": "1.5.0",
-    "@bufbuild/protoc-gen-es":"2.2.3",
+    "@bufbuild/protoc-gen-es": "2.2.3",
     "@microsoft/ccf-app": "5.0.14",
     "js-base64": "3.7.7",
     "node-forge": "1.3.1",
     "pem-jwk": "2.0.0"
   },
   "devDependencies": {
-    "@babel/core": "7.26.9",
-    "@babel/preset-env": "7.26.9",
-    "@babel/preset-typescript": "7.26.0",
     "@jest/globals": "29.7.0",
     "@rollup/plugin-commonjs": "28.0.3",
     "@rollup/plugin-node-resolve": "16.0.0",
     "@rollup/plugin-typescript": "12.1.2",
     "@types/jasmine": "5.1.5",
-    "@types/jest": "29.5.13",
     "@types/node": "22.13.10",
-    "axios": "1.8.2",
-    "babel-jest": "29.7.0",
-    "express": "4.21.1",
-    "jest": "29.7.0",
     "js-crypto-key-utils": "1.0.7",
     "js-crypto-rsa": "1.0.7",
-    "jsonwebtoken": "9.0.2",
-    "node-fetch": "3.3.2",
     "rollup": "4.35.0",
     "ts-jest": "29.2.6",
     "ts-node": "10.9.2",
-    "tslib": "2.7.0",
-    "typescript": "5.7.3",
-    "uglify-js": "3.19.3"
+    "tslib": "^2.8.1",
```

Comment:
we shouldn't use /\

Reply:
Thanks for that spot, I think it's a relic of the fact I removed it then re-added, so I'll restore it to the original state

Reply:
I actually noticed it works with a slightly newer version so just dropping the caret

https://github.com/microsoft/azure-privacy-sandbox-kms/pull/412/commits/cc8ed83723fd4762c816e3718b251877b3581350